app/image.html

```diff
@@ -0,0 +1,452 @@
+  <meta name="generator" content="Polymer Starter Kit" />
+  <title>DIG</title>
+  <!-- Place favicon.ico in the `app/` directory -->
+
+  <!-- build:css styles/main.css -->
+  <!-- Must link leaflet css (https://github.com/leaflet-extras/leaflet-map/issues/46) -->
+  <link rel="stylesheet" href="bower_components/leaflet/dist/leaflet.css">
+  <link rel="stylesheet" href="styles/main.css">
+  <!-- endbuild-->
+
+  <!-- build:js bower_components/webcomponentsjs/webcomponents-lite.min.js -->
+  <script src="bower_components/webcomponentsjs/webcomponents-lite.js"></script>
+  <!-- endbuild -->
+
+  <!-- will be replaced with elements/elements.vulcanized.html -->
+  <link rel="import" href="elements/elements.html">
+  <!-- endreplace-->
+
+  <style is="custom-style" include="entity-view-styles"></style>
+  <style is="custom-style" include="icon-styles"></style>
+  <style is="custom-style" include="icon-with-text-styles"></style>
+  <style is="custom-style" include="list-item-styles"></style>
+  <style is="custom-style" include="map-styles"></style>
+</head>
+
+<body unresolved class="fullbleed layout vertical">
+  <span id="browser-sync-binding"></span>
+  <template is="dom-bind" id="app">
+
+    <!-- exposes clientConfig object which is a container for data
+      transform functions -->
+    <client-config config="{{clientConfig}}"></client-config>
+
+    <!--get app configuration from server -->
+    <iron-ajax
+      id="ajax"
+      auto
+      url="/config"
+      handle-as="json"
+      last-response="{{config}}"
+      on-response="showConfig">
+    </iron-ajax>
+
+    <!--initialize new elastic client connection to server
+      esclient can be used throughout the application lifecycle -->
+    <elastic-client
+      config="[[config.elasticConfig]]"
+      client="{{esclient}}">
+    </elastic-client>
+
+    <!--look up and build an entity query for the entity specified in the
+      query string for:
+      field: the field in elasticsearch to use in the query
+      value: value of the field above -->
+
+    <!-- imageDisplay.image query and transform -->
+    <query-builder
+      query="{{imageQuery}}"
+      query-template="{{config.queryTemplates.commonMatchQuery}}"
+      field="[[params.field]]" value="[[params.value]]">
+    </query-builder>
+
+    <elastic-search
+      client="[[esclient]]"
+      index="[[config.elasticIndex]]"
+      query='[[imageQuery]]'
+      results="{{imageResults}}"
+      elastic-type='["image"]'
+      last-error="{{error}}">
+    </elastic-search>
+
+    <json-transform
+     data-in="[[imageResults]]"
+     data-out="{{imageDisplay}}"
+     transform-function="[[clientConfig.transforms.image.image]]"
+     ></json-transform>
+
+     <!-- Get the seller Object -->
+     <query-builder
+      query="{{sellerQuery}}"
+      query-template="{{config.queryTemplates.commonMatchQuery}}"
+      field="[[params.field]]" value="[[phoneDisplay.telephone.sellerId]]">
```

Comment:
There are references to phoneDisplay.telephone on lines 90, 116, and 220, but I don't think it's getting set anywhere.  Should those be references to imageDisplay instead?